website/docs/reference/node-selection/state-comparison-caveats.md

```diff
@@ -0,0 +1,99 @@
+---
+title: "Caveats to state comparison"
```

Comment:
Hmmm I don't feel like I'm in a good spot to review this doc — I feel a little far-removed from the nuances of this. Going to give it a 👍 but know that I didn't interrogate this too closely 😅